draft-ietf-quic-version-negotiation.md

```diff
@@ -405,25 +406,30 @@ VERSION_NEGOTIATION_ERROR (see {{iana-error}}).
 
 When a server receives a client's first flight, the server will first establish
 which QUIC version is in use for this connection in order to properly parse the
-first flight. For example, the server determines that QUIC version 1 is in use
+first flight. This may involve examining packets outside of the handshake, such
```

Comment:
I'm not sure what "examining packets outside of the handshake" means. INITIAL packets are still part of the handshake in the generic sense.

I suspect you meant something like "examining data that is not covered by the handshake transcript" but that's a mouthful and also quite specific to v1

How about just "This may involve examining the long header Version field."?

Reply:
Well, it might involve examining something else. E.g., the length of the packet.

Reply:
Agreed. But I still don't understand what "packets outside of the handshake" means

Reply:
PTAL